recipes/iqtree/build.sh

```diff
@@ -4,29 +4,46 @@ set -ex
 export INCLUDES="-I${PREFIX}/include"
 export LIBPATH="-L${PREFIX}/lib"
 export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
-export CFLAGS="${CFLAGS} -O3"
-export CPPFLAGS="${CPPFLAGS} -I${PREFIX}/include"
-export CXXFLAGS="${CXXFLAGS} -std=c++14"
+export CFLAGS="${CFLAGS} -O3 -w"
+export CPPFLAGS="${CPPFLAGS} -I${PREFIX}/include -w"
+export CXXFLAGS="${CXXFLAGS} -w"
 
 if [ "$(uname)" == Darwin ]; then
 	export CMAKE_C_COMPILER="clang"
 	export CMAKE_CXX_COMPILER="clang++"
+	export CXXFLAGS="${CXXFLAGS} -mmacosx-version-min=11"
+	CC=$(which "$CC")
```

Comment:
What's the idea here?

Reply:
Do you mean the minimum macos version or the use of which?

I found which was necessary to get the full path.

Reply:
I meant the usage of `which`.
Why a full path is needed ?
If `which` works then $CC is in $PATH

Reply:
Cmake seems to require the full path per my experiments, or maybe it's Ninja?

Happy to take out the witches and see if it still works!

Reply:
It doesn't work without `which`

```
08:37:02 BIOCONDA INFO (OUT) /bin/sh: $SRC_DIR/arm64-apple-darwin20.0.0-ar: No such file or directory
```
https://app.circleci.com/pipelines/github/bioconda/bioconda-recipes/123268/workflows/3f4c4171-720f-462b-b3b7-057088835e05/jobs/198401/parallel-runs/0/steps/0-107

Reply:
Weird! It assumes current dir ...
Oh, well!

Reply:
Maybe it's a ninja thing? Or cmake - who knows.